docs/adding_packages/dependencies.md

```diff
@@ -174,18 +174,14 @@ for consumer, we do impose some limits on Conan features to provide a smoother f
 * [`python_requires`](https://docs.conan.io/1/reference/conanfile/other.html#python-requires) are not allowed.
 
 ### Version Ranges
+ 
+Version ranges are a useful Conan feature, [documentation here](https://docs.conan.io/1/versioning/version_ranges.html). With the introduction of Conan 2.0, we are currently working to allow the use of version ranges and are allowing this for a handful of dependencies. Currently, these are:
+* OpenSSL: `[>=1.1 <4]` for libraries known to be compatible with OpenSSL 1.x and 3.x
+* CMake: `[>3.XX <4]`, where `3.XX` is the minimum version of CMake required by the relevant build scripts.
```

Comment:
`cmake/[^3.x.x]` is more elegant than `cmake/[>3.XX <4]`

Reply:
I also think that we should not constrain upper version of cmake. CMake is backward compatible.

Reply:
CMake itself is generally backwards compatible - but when CMake 3.0.0 was released, it broke compatibility with code that worked for CMake 2.4 or earlier - so it was a breaking release - it just so happens that it retained compatibility with 2.8 which was a long standing use.

I appreciate the version range we are proposing is perhaps too explicit and too careful, but if we were in CMake 2/3 times, a requirement on `cmake/[>=2.3]` would have caused failures when Conan started resolving to `cmake/3.0.0` upon release.  Hence we think, in absence of a explicit policy from CMake maintainers, stop at the boundary of the `major` component changing.

As for `cmake/[^3.x.x]`, we prefer to be explicit as @RubenRBS (and as @prince-chrismc  points out, it should be ` cmake/[>=3.x.x <4]`, which will be ammended).

Please note that PRs that do not adhere to these policies may not be considered for merge.